products/cockroachdb.md

```diff
@@ -0,0 +1,130 @@
+---
+title: CockroachDB
+category: database
+iconSlug: cockroachlabs
+permalink: /cockroachdb
+alternate_urls:
+-   /cockroach
+-   /crdb
+versionCommand: cockroach version
+releasePolicyLink: https://www.cockroachlabs.com/docs/releases/release-support-policy
+changelogTemplate: https://www.cockroachlabs.com/docs/releases/v__RELEASE_CYCLE__
+releaseDateColumn: true
+activeSupportColumn: Maintenance
+eolColumn: Assistance
+
+auto:
+  methods:
+  -   custom: cockroachdb
```

Comment:
```suggestion
  -   git: https://github.com/cockroachdb/cockroach.git
  -   release_table: https://www.cockroachlabs.com/docs/releases/release-support-policy
      selector: "table"
      headers_selector: "thead th"
      rows_selector: "tbody tr"
      fields:
        releaseCycle: "Version"
        releaseDate: "Release Date"
        support: "Maintenance Support ends"
        eol: "Assistance Support ends (EOL Date)"
```

See https://github.com/endoflife-date/release-data/pull/319#issuecomment-1965329009. Also added the git configuration so that latest releases are automatically updated.